scripts/install_runtime_dependencies.sh

```diff
@@ -1,19 +1,21 @@
 #!/bin/bash
 
-BUILD_DEPENDS=(gnupg2 gpg)
+BUILD_DEPENDS=(gpg)
 
 apt-get update
 apt-get install -y ${BUILD_DEPENDS[@]} curl libdlib19.1 ffmpeg exiftool libheif1
 
 # Install Darktable if building for a supported architecture
 if [ "${TARGETPLATFORM}" = "linux/amd64" ] || [ "${TARGETPLATFORM}" = "linux/arm64" ]; then
-  echo 'deb https://download.opensuse.org/repositories/graphics:/darktable/Debian_12/ /' \
-    | tee /etc/apt/sources.list.d/graphics:darktable.list
+  echo 'deb https://download.opensuse.org/repositories/graphics:/darktable/Debian_12/ /' > /etc/apt/sources.list.d/darktable.list
   curl -fsSL https://download.opensuse.org/repositories/graphics:/darktable/Debian_12/Release.key \
-    | gpg --dearmor | tee /etc/apt/trusted.gpg.d/graphics_darktable.gpg > /dev/null
+    | gpg --dearmor -o /etc/apt/trusted.gpg.d/darktable.gpg
+  gpg --show-keys --with-fingerprint --dry-run /etc/apt/trusted.gpg.d/darktable.gpg
```

Comment:
Is this a debug code, which you forgot to remove?)

Reply:
I hope to keep it as a building log.